calvinpete/Store-Manager-react
# #164600549 Deploy on Heroku

### What does this PR do?
This PR deploys our application to heroku

### Description of Task to be completed?
- add server.js file
- add Procfile
- Modify package.json

### How should this be manually tested?
- fetch the branch using `git fetch origin ch-deploy-heroku-164600549`
- check into the branch using `git checkout ch-deploy-heroku-164600549`
- run `npm install ` to install the dependencies
- run `npm run build` to start the development server
- navigate to https://store-manager-react17.herokuapp.com/

### What are the relevant pivotal tracker stories?
[#164600549](https://www.pivotaltracker.com/story/show/164600549)
